### PR TITLE
Accept more extensions when completing for -U in fish

### PR DESCRIPTION
**Pull request by sboukortt**
Fixes issue #49.

### CODE REVIEW

File: packaging/usr/share/fish/completions/pikaur.fish [hunk 1]
```diff
@@ -167,7 +167,8 @@ complete -c $progname -n "$files" -l machinereadable -d 'Show in machine readabl
 
 # Upgrade options
 # Theoretically, pacman reads packages in all formats that libarchive supports
-# In practice, it's going to be tar.xz or tar.gz
+# In practice, it's going to be tar.xz, tar.gz or tar
 # Using "pkg.tar.*" here would change __fish_complete_suffix's descriptions to "unknown"
 complete -c $progname -n "$upgrade" -xa '(__fish_complete_suffix pkg.tar.xz)' -d 'Package file'
 complete -c $progname -n "$upgrade" -xa '(__fish_complete_suffix pkg.tar.gz)' -d 'Package file'
+complete -c $progname -n "$upgrade" -xa '(__fish_complete_suffix pkg.tar)'    -d 'Package file'
```
Comment (actionless) [inline packaging/usr/share/fish/completions/pikaur.fish]: if it's working if you replace all 3 lines to 
```fish
complete -c $progname -n "$upgrade" -xa '(__fish_complete_suffix "pkg.tar*")'    -d 'Package file'
```
?

Comment (sboukortt) [reply packaging/usr/share/fish/completions/pikaur.fish]: Yes, for me it does. Should I update the PR?

Comment (actionless) [reply packaging/usr/share/fish/completions/pikaur.fish]: in that case yes

Comment (sboukortt) [reply packaging/usr/share/fish/completions/pikaur.fish]: Done.

**Comment by actionless** [comment]
Thanks!

**Pull request merged**